BardMusicPlayer/UI_Classic/Classic_MainViewPlaycontrols.cs

```diff
@@ -42,6 +44,22 @@ private void Play_Button_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        private void Play_Button_MouseRightButtonDown(object sender, System.Windows.Input.MouseButtonEventArgs e)
+        {
+            if (PlaybackFunctions.PlaybackState == PlaybackFunctions.PlaybackState_Enum.PLAYBACK_STATE_PLAYING)
+                return;
+
+            if (!BmpPigeonhole.Instance.UsePluginForInstrumentOpen)
+                return;
+
+            Task task = Task.Run(() =>
+            {
+                BmpMaestro.Instance.EquipInstruments();
+                Task.Delay(2000).Wait();
```

Comment:
This is bad, you can't just assume everyone equipped an instrument on a static timer.